css/src/components/layout.scss

```diff
@@ -229,6 +338,36 @@ $three-quarters-widescreen: math.div($breakpoint-widescreen, 4) * 3;
 	-webkit-overflow-scrolling: touch;
 }
 
+.layout-body-flyout-sticky {
+	@include constrained-layout-child;
+}
+
+@include discouraged-mobile {
```

Comment:
Would it be simpler to put `display: none;` on `layout-body-flyout` as default, then show it with `.layout-flyout-active` as desktop, rather than using both `discouraged-mobile` and `tablet` mixin to hide it on mobile and tablet ?

Reply:
Yes, it would definitely be simplier. My reasoning here was that I didn't want to set/assign an particular display value on desktop, but I may have just traded one problem for another. I'll update